runtime/moonbase/src/lib.rs

```diff
@@ -452,18 +452,30 @@ impl pallet_democracy::Config for Runtime {
 parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = 1 * currency::UNITS;
-	pub const SpendPeriod: BlockNumber = 6 * DAYS;
+	pub const SpendPeriod: BlockNumber = 10;
```

Comment:
This is a big change in terms of time. You want it to be 10 blocks right? not `10 * DAYS`?

Reply:
We need a short time on the test network to test that the recipient is receiving the funds.

Reply:
`moonbase` is the runtime for our public Alphanet, so we shouldn't modify those value if not in the purpose to improve the network.

Reply:
Ok I revert this diff.